src/languages/en.ts

```diff
@@ -1840,6 +1840,9 @@ export default {
             selectSuggestedAddress: 'Please select a suggested address or use current location',
         },
     },
+    eReceipt: {
```

Comment:
I don't think we need this block as it already exists on L1850,=,

Reply:
Oh woops, thanks.